.github/workflows/build.yml

```
@@ -11,17 +11,30 @@ concurrency:
   UNIT_TESTS_COVERAGE_REPORTS_PATH: tests/unit/coverage
+  UNIT_TESTS_COVERAGE_BASE_CACHE_KEY: tests-unit-coverage-${{ needs.init.outputs.safe_branch_name }}
   ACCEPTANCE_TESTS_REPORTS_PATH: tests/acceptance/reports
   GH_TOKEN: ${{ github.token }}
   STRYKER_REPORT_PATH: tests/stryker/coverage
   STRYKER_INCREMENTAL_PATH: tests/stryker/incremental
 
 jobs:
+  init:
+    runs-on: ubuntu-latest
+    outputs:
+      safe_branch: ${{ steps.sanitize-branch.outputs.safe_branch }}
```

actionlint (1.7.7)

property "sanitize-branch" is not defined in object type {sanitize-branch-name: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)